kisanola/tech-community-frontend
# Added Scroll Back To Top

#### What does this PR do?
This PR adds a button to scroll to the top of the page.

#### Description of Task to be completed?
Add a button to scroll to the top
If at the top, hide the button

#### How should this be manually tested?
Fetch and checkout to `ft-scroll-to-top`
Run the server with an active internet connection
Scroll down and the button to `Scroll To Top` will appear
Click the button and will it you to the top and dissappear.

#### Screenshots (if appropriate)
![Screenshot from 2020-04-26 14-42-41](https://user-images.githubusercontent.com/37688326/80306495-61ac9580-87cc-11ea-8335-faaed90f0e1a.png)
![Screenshot from 2020-04-26 14-42-48](https://user-images.githubusercontent.com/37688326/80306498-65d8b300-87cc-11ea-82b1-623abb1b8e7d.png)

#### Questions:
